storage/ndb/src/kernel/vm/mt.cpp

```diff
@@ -8067,76 +8054,46 @@ mt_add_thr_map(Uint32 block, Uint32 instance)
   case DBTUX:
   case BACKUP:
   case RESTORE:
-    if (receive_threads_only)
-    {
-      thr_no += (instance - 1);
-    }
-    else
-    {
-      thr_no += (instance - 1);
-    }
+    thr_no += (instance - 1);
```

Comment:
These blocks are handled by ldm threads previously. Should we handle the situation that (ldm threads num == 0 && tc threads num > 0) here?

Reply:
isn't allowed to happen that num_ldm_threads == 0 AND num_tc_threads > 0